src/internal/plugins/controllers/drawers.ts

```diff
@@ -30,6 +35,7 @@ export interface DrawerConfig {
   mountContent: (container: HTMLElement, mountContext: MountContentContext) => void;
   unmountContent: (container: HTMLElement) => void;
   preserveInactiveContent?: boolean;
+  onStateChange?: NonCancelableEventHandler<DrawerStateChangeParams>;
```

Comment:
Let's change it to `onToggle`:

```
  onToggle: (event: { isOpen: boolean; isUserAction: boolean;  }) => {
   // defaultActive application – false
   // toggle click - true
   // limit of 3 open exceeded - true
   // limit because screen width - true
  },
```